services.yaml

```diff
@@ -383,9 +383,10 @@ paths:
           type: string
           required: true
         - in: query
-          name: active
-          description: Show only active deployments
-          type: boolean
+          name: state
```

Comment:
this is a breaking change i am not sure we should do. 
we can add it alongside active as a newer feature

Reply:
fixed